test/runtests.jl

```diff
@@ -20,6 +20,12 @@ using ZMQ, Test
 
     #try to create socket with expired context
     @test_throws StateError Socket(ctx, PUB)
+
+    # Smoke tests for Base.show()
+    ctx = Context()
+    show(IOBuffer(), ctx)
+    close(ctx)
+    show(IOBuffer(), ctx)
```

Comment:
Can't you do
```
@test repr(ctx) == "Context([closed])"
```
or similar?  (`repr` calls `show`)

Reply:
Ah cool, didn't realize that. Fixed in a95c73b.